CITATION.cff

```diff
@@ -0,0 +1,14 @@
+cff-version: 1.2.0
+message: "If you use this software, please cite it as below."
+authors:
+- family-names: "Maurya"
```

Comment:
hi @sugatoray, thank you for your contribution 😄 
Is it possible to update this automatically? I guess authors would increase in future.

Reply:
@aniketmaurya I guess you could for now set a 3/6 monthly schedule to revisit and determine who could be included in the list of authors. 

So far automation is concerned, you can find GitHub actions to add list of contributors to a given file automatically. Perhaps you could use that to think of downstream automation.